### PR TITLE
Update Create Image notifications

### PR DESCRIPTION
**Pull request by jgiardino**
- Removes previous notifications. These weren't accurately reflecting the image creation process.
- Adds notification text for each status type
- Only displays notification for status 'imageWaiting'. The other notifications will be added in a later PR.

### CODE REVIEW

File: data/NotificationsApi.js [hunk 1]
```diff
@@ -32,32 +32,39 @@ class NotificationsApi {
...
+        };
+        break;
+      }
+      case 'imageStarted': {
+        notification = {
+          type: 'success',
           message: <span>
             <FormattedMessage
-              defaultMessage="{blueprint} Creating image."
+              defaultMessage="{blueprint} Image creation has started."
```
Comment (henrywang) [inline data/NotificationsApi.js]: One failed case is due to this change. Just update the 22nd line of `pages/toastNotif.js`.
The ```this.varStatusCreating = `${this.varBlueprintName} Creating image.`;``` should be updated to ```this.varStatusCreating = `${this.varBlueprintName} Image creation has started.`;```

**Comment by henrywang** [comment]
@jgiardino, the notification workflow changed, but just got notification with  "Image creation has been added to the queue." message. No more notification found in my local environment. I think there should have notification when image creation failed or success.

It has to update two files to make test work again.
1. spec/test_viewBlueprint.js
Please remove lines from 155-159, because there's no second status in notification workflow.
```
            browser
              .waitForVisible(toastNotifPage.iconComplete);

            const textComplete = $(toastNotifPage.labelStatus).getText();
            assert.equal(textComplete, toastNotifPage.varStatusComplete);
```
2. pages/toastNotif.js
`Line 13`: update to ```this.iconCreating = '#cmpsr-toast-0 .pficon-info';```
`Line 22`: update to ```this.varStatusCreating = `${this.varBlueprintName} Image creation has been added to the queue.`;```

**Comment by jgiardino** [comment]
Thanks for the suggestions, @henrywang! I made these updates and the tests passed locally.

**Comment by jgiardino** [comment]
I forgot to comment on this:

> I think there should have notification when image creation failed or success.

Yes, I agree that we should display notifications for each status change in the image build process. For this PR, I just wanted to remove the notifications that weren't accurately reflecting what's happening with the build process. Also, I can easily display a notification for when a build is added to the queue, but the other status changes will require more work and will be handled in a separate PR.

**Comment by henrywang** [comment]
@jgiardino, thanks for your explanation, it makes sense to me.

**Pull request merged**